### DISCUSSION TITLE
有没有什么简便的方法在yun主题中加入一个文档页面？

### DISCUSSION THREAD

**Discussion by SukiYume** [YunYouJun/valaxy#363] [General]
请教一下，我想在posts、tags、archives同一级目录中添加一个docs文件夹，只让docs下面的内容按照类似press主题中[guide](https://valaxy.site/guide)页面的方式进行渲染。就是当访问/docs页面时，左侧的sidebar按照文件夹结构组织链接，以快速访问/docs/aaa/01、/docs/bbb/02之类的链接。请问有没有什么方法来实现？非常感谢！

```txt
pages
├─ posts
├─ tags
├─ docs
│  ├─ aaa
|     ├─ Figure
|     ├─ 01.md
│     └─ 02.md
|  ├─ bbb
|     ├─ Figure
|     ├─ 01.md
|     └─ 02.md
```

**Comment by YunYouJun** [comment]
新建 `layouts/docs`，然后自定义布局即可。

Markdown 使用布局：

```md
---
layout: docs
---
```

你可以直接复用 ThemeYun 中已有的组件。

**Reply by SukiYume** [reply]
感谢！我第一次接触vue，可能要慢慢摸索着写一写试一试了。
